scripts/ci/qemu-self-test-seabios.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+# Define an array of commands
+commands=(
+  "robot -X -L TRACE -v BIOS_LIB:seabios -v config:qemu-selftests-seabios -v rte_ip:127.0.0.1 -d ./logs/$(date +%Y.%m.%d_%H.%M.%S)/setup-and-boot-seabios -v snipeit:no self-tests/setup-and-boot-seabios.robot"
```

Comment:
BIOS_LIB can be set in platform config

Reply:
fixed in https://github.com/Dasharo/open-source-firmware-validation/commit/e383a2555926adf42b2cf5838c65d640e76cc204